prusti-viper/src/verifier.rs

```diff
@@ -319,8 +320,19 @@ impl<'v, 'tcx> Verifier<'v, 'tcx> {
 
             for verification_error in verification_errors {
                 debug!("Verification error: {:?}", verification_error);
-                let prusti_error = error_manager.translate_verification_error(&verification_error);
+                let mut prusti_error = error_manager.translate_verification_error(&verification_error);
                 debug!("Prusti error: {:?}", prusti_error);
+                
+                //counterexamples:
+                if let Some(id) = error_manager.get_def_id(&verification_error) {
+                    let counterexample = backtranslation::backtranslate(
+                        &self.encoder, 
+                        *id,
+                        verification_error.counterexample
+                    );
+                    counterexample.apply_prusti_error(&mut prusti_error); 
+                }
```

Comment:
I would have expected the `config` to be checked here?

Reply:
I guess that's saver than just relying on Silicon to only produce one when we set the flags. Should I also add this to the counterexample extraction in viper/verifier.rs? Or is that a bad idea because prusti_common is currently not used in that crate?